.gitignore

```diff
@@ -0,0 +1 @@
+*/__pycache__/
```

Comment:
we can avoid the `.gitignore` files in the repo if you set the entries locally in your global git config. no big deal though.